src/com/iandonaldson/data/LocationDaoImpl.java

```diff
@@ -1,47 +1,86 @@
 package com.iandonaldson.data;
-
 import java.sql.Connection;
+import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.sql.Statement;
+import java.util.LinkedList;
+import java.util.List;
 
 public class LocationDaoImpl implements LocationDao {
 	public LocationDaoImpl() {
 		
 	}
 	
-	public Location getLocationFromDB(String query) {
-		// TODO make able to use PREPAREDsTATMENT as function parameters
+	public Location getLocationFromDB(PreparedStatement ps) {
```

Comment:
Not sure why you called this method getLocationFromDB. I would just call it getLocation. Remember, think of them as verbs with no qualifier such as DB. There are differences such as getActorsByFilm, but that's a descriptive qualifier on the verb.